### DISCUSSION TITLE
bot stream video on call!

### DISCUSSION THREAD

**Discussion by sebastianjnuwu** [discordjs/discord.js#7969] [Ideas]
I didn't find anything about the bot broadcasting in call videos or something like that I see it would be a good one lol

**Comment by vvito7** [comment]
See https://github.com/discord/discord-api-docs/issues/1995#issuecomment-678550660.

**Reply by sebastianjnuwu** [reply]
humm

**Reply by vvito7** [reply]
Basically, there is no way to discord.js to implement a feature that doesn't exists (the reason is in the comment that I shared).
